build.gradle

```diff
@@ -4,7 +4,13 @@ plugins {
 
 allprojects {
     project.group = "io.honeycomb"
-    project.version = "0.8.1"
+    def tag = System.getenv("CIRCLE_TAG")
```

Comment:
It would be nicer if we didn't have duplicated version strings here 😢

Reply:
#249 249